src/main/java/org/ow2/proactive/connector/iaas/cloud/provider/azure/AzureServiceBuilder.java

```diff
@@ -98,10 +99,14 @@ private AzureEnvironment getAzureEnvironment(Infrastructure infrastructure) {
         Optional<String> optionalGraphEndpoint = Optional.ofNullable(infrastructure.getGraphEndpoint());
         if (optionalAuthenticationEndpoint.isPresent() && optionalManagementEndpoint.isPresent() &&
             optionalResourceManagerEndpoint.isPresent() && optionalGraphEndpoint.isPresent()) {
-            return new AzureEnvironment(optionalAuthenticationEndpoint.get(),
-                                        optionalManagementEndpoint.get(),
-                                        optionalResourceManagerEndpoint.get(),
-                                        optionalGraphEndpoint.get());
+            HashMap<String, String> customAzureEnvironment = new HashMap<>();
+            //customAzureEnvironment.put(AzureEnvironment.Endpoint.ACTIVE_DIRECTORY.toString(), optionalAuthenticationEndpoint.get());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)